### PR TITLE
fix(jsx-indent): conflict between indent and jsx-indent

### PR DESCRIPTION
**Pull request by Daydreamer-riri**
### Description

Currently, there are some issues with jsx-indent, where some indentations are not checked, and there are conflicts with the indent rules. I have added some tests that cannot be passed.

### Additional context

image
image
image
image

### CODE REVIEW

**Comment by antfu** [comment]
The test seems to fail, can you take a look? Thanks

**Comment by Daydreamer-riri** [comment]
Apologies, I realized I overlooked a few things. When I set `style/indent` (which I use with `@antfu/eslint-config`) to `'style/indent': ['error', 2],` most of the issues I raised disappeared. It seems that JSX-related indentation is ignored by default in `@antfu/eslint-config`? I'm curious to know why this is the case.

I only have this one question left at the moment. I'm not sure if this is still due to something I've overlooked.
image

**Comment by Daydreamer-riri** [comment]
Hi @antfu , I would like to know if there is any specific reason why the indent rule defaults to ignore JSX-related nodes. This will determine whether I should implement indentation within the JSXExpressionContainer in the jsx-indent rule.

**Comment by Daydreamer-riri** [comment]
This will fix #312 .

**Comment by Daydreamer-riri** [comment]
This PR is ready.

File: packages/eslint-plugin-jsx/rules/jsx-indent/jsx-indent.ts [hunk 1]
```diff
@@ -380,6 +380,8 @@ export default createRule<MessageIds, RuleOptions>({
       const nameIndent = getNodeIndent(node.name)
       const lastToken = context.sourceCode.getLastToken(node.value)!
       const firstInLine = getFirstNodeInLine(context, lastToken)
+      if (firstInLine.loc.start.line !== lastToken.loc.start.line)
```
Comment (antfu) [inline packages/eslint-plugin-jsx/rules/jsx-indent/jsx-indent.ts:383]:
```suggestion
      // leave this for `indent` rule to handle
      if (firstInLine.loc.start.line !== lastToken.loc.start.line)
```

**Review by antfu — APPROVED** [review]

**Pull request merged**